lib/msal-node/src/config/Configuration.ts

```diff
@@ -107,6 +111,11 @@ export type Configuration = {
     telemetry?: NodeTelemetryOptions;
 };
 
+export type ManagedIdentityConfiguration = Omit<
+    Configuration,
+    "auth" | "broker" | "cache" | "telemetry"
```

Comment:
Does MSAL Node support any telemetry other than server side telemetry?

Reply:
yes, it supports client telemetry. I will need to implement that eventually

Reply:
Actually I don't think you need to do that @Robbie-Microsoft . The way public client and SPA **client** telemetry is collected is different than confidential client. @neha-bhargava will do a demo on this soon I've heard :)